docs/configuration/communication/vault-csi.md

```diff
@@ -44,7 +44,8 @@ This instruction guides you through the installation of Botkube and Vault on a K
 
    ```bash
    # Write the token to Vault
-   vault kv put secret/slack token={token}
+   vault kv put -mount=secret slack-app-token token={xapp-...}
+   vault kv put -mount=secret slack-bot-token token={xoxb-...}
```

Comment:
Just to make it clear, I would suggest going in one or another direction:

Option 1:
```suggestion
   vault kv put -mount=secret slack-app-token token=xapp-...
   vault kv put -mount=secret slack-bot-token token=xoxb-...
```

Option 2:
```suggestion
   vault kv put -mount=secret slack-app-token token={appToken}
   vault kv put -mount=secret slack-bot-token token={botToken}
```

Whatever you prefer 🙂